data/reusables/actions/actions-group-concurrency.md

```diff
@@ -1,4 +1,6 @@
-When a concurrent job or workflow is queued, if another job or workflow run using the same concurrency group in the repository is in progress, the queued job or workflow run will be `pending`. Any pending job or workflow run in the same concurrency group will be canceled when a new job or workflow run is queued. To also cancel any currently running job or workflow run in the same concurrency group, specify `cancel-in-progress: true`.
+When a concurrent job or workflow is queued, if another job or workflow using the same concurrency group in the repository is in progress, the queued job or workflow will be `pending`. Any pending job or workflow in the concurrency group will be canceled.  This means that there can be at most 1 running and 1 pending job in a concurrency group at any one time.
+
+To also cancel any currently running job or workflow in the same concurrency group, specify `cancel-in-progress: true`.  You may also specify `cancel-in-progress` as an expression subject to the restrictions on available contexts outlined above.
```

Comment:
```suggestion
To also cancel any currently running job or workflow in the same concurrency group, specify `cancel-in-progress: true`.  To conditionally cancel currently running jobs or workflows in the same concurrency group, you can specify `cancel-in-progress` as an expression if you use any of the following allowed expression contexts: [`github`](/actions/learn-github-actions/contexts#github-context), [`inputs`](/actions/learn-github-actions/contexts#inputs-context), [`vars`](/actions/learn-github-actions/contexts#vars-context), [`needs`](/actions/learn-github-actions/contexts#needs-context), [`strategy`](/actions/learn-github-actions/contexts#strategy-context), and [`matrix`](/actions/learn-github-actions/contexts#matrix-context).
```
We try to refrain from saying things like "outlined above" in reusables, because if the reusable is placed in a doc where the thing isn't outlined above, that would be really confusing. So I just repeated the information here.